InfiniteRecharge/src/main/java/frc/robot/RobotMap.java

```diff
@@ -34,12 +37,27 @@ public static SensoredSystem getRobotWranglerSystem() {
         
         SensoredSystem system = new SensoredSystem(robotWranglerMotor);
         return system;
-    }
+	}
+	
+	public static SensoredSystem getControlPanelSystem() {
+		PbTalonSrx controlPanelMotor;
+		controlPanelMotor = new PbTalonSrx(Constants.kControlPanelMotor);
+		SensoredSystem system = new SensoredSystem(controlPanelMotor);
+		return system;
+	}
 
     public static LimitSwitchGroup getRobotWranglerLimits() {
         PbDioSwitch robotWranglerForwardLimit = new PbDioSwitch(Constants.kRobotWranglerForwardLimitDio);
         PbDioSwitch robotWranglerReverseLimit = new PbDioSwitch(Constants.kRobotWranglerReverseLimitDio);
         LimitSwitchGroup limitSwitchGroup = new LimitSwitchGroup(robotWranglerForwardLimit, robotWranglerReverseLimit);
         return limitSwitchGroup;
-    }
+	}
+	public static PbSolenoid getControlPanelSolenoid() {
+		if(Constants.kControlPanelDoubleSolenoid) {
+		PbSolenoid controlPanelSolenoid = new PbSolenoid(Constants.kPcmCanID, Constants.kControlPanelSolenoidAChannel,
+		 Constants.kControlPanelSolenoidBChannel);
+		} else {
+		PbSolenoid controlPanelSolenoid = new PbSolenoid(Constants.kPcmCanID, Constants.kControlPanelSolenoidAChannel);}
+		return getControlPanelSolenoid();
+	} 
```

Comment:
Please change this to return a normal solenoid and change the declaration to match bypassing the if statement.